packages/components/src/navigator/styles.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { css, keyframes } from '@emotion/react';
+
+export const navigatorProviderWrapper = css`
+	/* Prevents horizontal overflow while animating screen transitions */
+	overflow-x: hidden;
+
+	contain: strict;
+`;
+
+const fadeInFromRight = keyframes( {
+	'0%': {
+		opacity: 0,
+		transform: `translateX(50px)`,
+	},
+	'100%': { opacity: 1, transform: 'none' },
+} );
+
+const fadeInFromLeft = keyframes( {
+	'0%': {
+		opacity: 0,
+		transform: `translateX(-50px)`,
```

Comment:
Spacing nits. Our linter seems to ignore spacing in that context.

```suggestion
		transform: `translateX( 50px )`,
	},
	'100%': { opacity: 1, transform: 'none' },
} );

const fadeInFromLeft = keyframes( {
	'0%': {
		opacity: 0,
		transform: `translateX( -50px )`,
```

Reply:
https://github.com/WordPress/gutenberg/pull/56909/commits/9fbbc088a05df64510cc2dbc2aa80152573267a1